tests/integration_tests/data_sources/test_maria_ds.py

```diff
@@ -0,0 +1,62 @@
+import mysql.connector
+import datetime
+import logging
+from mindsdb import Predictor
+from mindsdb.libs.constants.mindsdb import DATA_TYPES, DATA_SUBTYPES
+from mindsdb.libs.data_sources.maria_ds import MariaDS
+
+
+def test_maria_ds():
+    HOST = 'localhost'
+    USER = 'root'
+    PASSWORD = ''
+    DATABASE = 'mysql'
```

Comment:
Is this the default database for mariadb ?

Reply:
Seems like it